VideoWeb/VideoWeb/ClientApp/src/app/shared/models/judge-hearing-summary.spec.ts

```diff
@@ -65,13 +66,15 @@ describe('JudgeHearingSummary', () => {
         const participants = hearing.nonJudicialParticipantsExcludingObservers;
 
         expect(
-            participants.filter(x => x.role !== Role.Individual && x.role !== Role.Representative && x.hearingRole !== HearingRole.WINGER)
-                .length
+            participants.filter(
+                x =>
+                    x.role !== Role.Individual &&
+                    x.role !== Role.Representative &&
+                    x.hearingRole !== HearingRole.WINGER &&
+                    x.role !== Role.QuickLinkParticipant
```

Comment:
QuickLinkObserver ?

Reply:
As mentioned in my other comment quicklink observers will not be included in this function.